dbt_copilot_helper/commands/bootstrap.py

```diff
@@ -94,6 +102,7 @@ def make_config(directory="."):
         )
         name = service["name"]
         (base_path / f"copilot/{name}/addons/").mkdir(parents=True, exist_ok=True)
+        _generate_svc_overrides(base_path, templates, name)
```

Comment:
Not something to address here, but what about completely new applications not created with a bootstrap file.

Reply:
yep will create a new ticket to pick that up